content/community_calls/2025-10-15 Notes.en.md

```
@@ -0,0 +1,37 @@
+weight: 967
+type: "post"
+date: 2025-10-15
+---
+
+### Recording
+
+{{< youtube id="_ezyrvAVszc" >}}
+
+## Agenda
+
+#### Community presentation
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)